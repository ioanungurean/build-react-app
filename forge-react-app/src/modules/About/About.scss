@import 'xbem.scss';

@include block(about) {}
