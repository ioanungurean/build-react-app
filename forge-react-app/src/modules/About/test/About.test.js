import React from 'react';
import { shallow } from 'enzyme';

import About from '../About';

describe('<About />', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});
