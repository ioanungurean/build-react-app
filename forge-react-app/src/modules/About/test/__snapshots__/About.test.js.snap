// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render without throwing an error 1`] = `
<div
  className="about"
>
  About
</div>
`;
